src/dp_periodic_msg.c

```diff
@@ -15,6 +15,11 @@
 static uint8_t dp_mc_ipv6[16] = { 0xff, 0x02, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0x01 };
 static uint8_t dp_mc_mac[6] = { 0x33, 0x33, 0x00, 0x00, 0x00, 0x01 };
 
+static __rte_always_inline bool dp_is_ip_set(struct dp_port *port, uint16_t eth_type)
+{
+	return (eth_type == RTE_ETHER_TYPE_IPV6 && !dp_is_ipv6_addr_zero(port->iface.cfg.dhcp_ipv6)) ||
+		   (eth_type == RTE_ETHER_TYPE_ARP && port->iface.cfg.own_ip != 0);
+}
```

Comment:
I am not sure I understand the condition now (or there is a typo). 
IP is set when type is IPV6 and non zero (sure), OR when type is ARP *and* ipv4 is not zero?

Reply:
If you check the callers of `send_to_all_vfs()` then you will see that the function is actually called to send ARP and ND packets periodically to "all" VFs. (It is only called for ether type ARP and ether type IPv6) 
In case of ipv4 there is the ARP ether type and in case of IPv6, it is the ND packet which is ether type IPv6.
The previous use of ether type IPv4 was not correct so completely removed.

Reply:
Well, this is the condition which we want to omit in the loop where this is called maybe the function name needs to change to smt like `dp_is_port_ok_to_send()`

Reply:
Can you put the summary of your first explanation into a comment there? I get it now, but without your explanation it is unclear from looking at the function.

Reply:
Done